src/penn_chime/view/st_app.py

```diff
@@ -31,6 +31,10 @@ def main():
     d = Parameters.create(os.environ, [])
     p = display_sidebar(st, d)
     m = Sir(p)
+    
+    if not m.reasonable_model_parameters:
+        st.subheader("Pre-mitigation doubling time is greater than 15 days.\nTry different parameters.")
```

Comment:
I think it would be better to phrase this a bit more explicitly, along the lines of

"Based on the current parameters, the estimated pre-mitigation doubling time is ..."

Reply:
I added this :)